### PR TITLE
Db cleanup

### PR DESCRIPTION
**Pull request by licode**
Here we still use old databroker. I clean up all the wrong use of db, and store all the databroker related import at 00-startup only. So we can easily upgrade databroker to 0.9.0+ soon.

### CODE REVIEW

**Comment by licode** [comment]
The two databroker wrapper is just a temp solution for now.

**Comment by licode** [comment]
@stuartcampbell

**Comment by stuartcampbell** [comment]
Cool - thank you.  I would merge it, but I don't have permissions! 😢

**Comment by licode** [comment]
No worry, we can leave it here for a few days. HXN wants to do more tests on this.

File: profile_collection/startup/00-startup.py [hunk 1]
```diff
@@ -29,6 +29,77 @@
...
+    def __getitem__(self, key):
+        try:
+            return db_new[key]
+        except ValueError:
+            return db_old[key]
+
+    def get_table(self, *args, **kwargs):
+        result = db_new.get_table(*args, **kwargs)
+        if len(result) == 0:
+            result = db_old.get_table(*args, **kwargs)
```
Comment (danielballan) [inline profile_collection/startup/00-startup.py]: I think we want the old results even if the new results are not empty. For example, if I search for all runs where `plan_name='count'`, I should get back the union of the new and the old.

File: profile_collection/startup/00-startup.py [hunk 2]
```diff
@@ -29,6 +29,77 @@
 pd.options.display.max_columns = 10
 
 
+from metadatastore.mds import MDS
+from databroker import Broker
+from databroker.core import register_builtin_handlers
+from filestore.fs import FileStore
+
+_mds_config = {'host': 'xf03id-ca1',
```
Comment (danielballan) [inline profile_collection/startup/00-startup.py]: This is a step forward, and could be merged as is, but we are moving toward using yml files like `/etc/databroker/hxn.yml`. We laid out the provision plan for doing this [here](https://github.com/NSLS-II/databroker/issues/194) and documented the final version [here](https://nsls-ii.github.io/databroker/configuration.html).

File: profile_collection/startup/00-startup.py (hunk 1 above)
Comment (licode) [reply profile_collection/startup/00-startup.py]: good point.

**Comment by licode** [comment]
I merge it for now. I will send a new pr with latest databroker api.

**Pull request merged**